akka-http-bench-jmh/src/main/scala/akka/http/impl/engine/http2/H2ServerProcessingBenchmark.scala

```diff
@@ -77,18 +81,10 @@ class H2ServerProcessingBenchmark extends CommonBenchmark {
         .withFallback(ConfigFactory.load())
     system = ActorSystem("AkkaHttpBenchmarkSystem", config)
     mat = ActorMaterializer()
-    val settings = implicitly[ServerSettings]
-    val log = system.log
     implicit val ec = system.dispatcher
-    val http1 = Flow[SslTlsInbound].mapAsync(1)(_ => {
-      Future.failed[SslTlsOutbound](new IllegalStateException("Failed h2 detection"))
-    })
-    val http2 =
-      Http2Blueprint.handleWithStreamIdHeader(1)(req => {
-        req.discardEntityBytes().future.map(_ => response)
-      })(system.dispatcher)
-        .join(Http2Blueprint.serverStackTls(settings, log, NoOpTelemetry))
-    httpFlow = Http2.priorKnowledge(http1, http2)
+    httpFlow = Http2Blueprint.handleWithStreamIdHeader(1)(req => {
```

Comment:
`handleWithStreamIdHeader` doesn't do anything, so you basically removed the whole HTTP/2 pipeline here. Was that intended?

Reply:
oops, no :S